scripts/test.sh

```diff
@@ -22,7 +22,7 @@
 
 set -euo pipefail
 
-GO111MODULE=on ginkgo -r \
+GO111MODULE=on ginkgo ./... -r \
```

Comment:
Out of curiosity, was that added by GoLand?

Reply:
No I added it, as for whatever reason ginkgo kept failing without a specified path

Reply:
Works now, removed it again